{"name":"GetByUserID: Failure","fullName":"TestManagerPgRepoSuiteRunner/ManagerPgRepoSuite/TestRequestPgRepo_GetByUserIDFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1728207864920,"stop":1728207864921,"uuid":"925f0626-83c7-11ef-9255-be2bab8c44a2","historyId":"b4301e8cbd841d27b7e53d0c179ce9eb","testCaseId":"299c0e2e5259d0bfc5587c5de2da5177","labels":[{"name":"language","value":"go1.23.0"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MBP-Pavel-2.netis.cc"},{"name":"thread","value":"TestManagerPgRepoSuiteRunner/ManagerPgRepoSuite/TestRequestPgRepo_GetByUserIDFailure"},{"name":"suite","value":"ManagerPgRepoSuite"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"ManagerPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1728207864920,"stop":1728207864921,"steps":[{"name":"ASSERT: Nil","status":"passed","start":1728207864921,"stop":1728207864921,"parameters":[{"name":"Actual","value":"*models.Manager((*models.Manager)(nil))"}]},{"name":"ASSERT: Error Is","status":"passed","start":1728207864921,"stop":1728207864921,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}